README.md

```diff
@@ -43,8 +43,10 @@ In particular,
 - Thinking beyond simple circuits will allow us to do new things and progress the field.
 
 Compilation and optimization of such programs is crucial to support quantum hardware execution,
-however current de-facto interchange formats (such as QIR and OpenQASM) either aren't designed for quantum optimization — lacking the explicit
-structure needed for advanced high-level optimizations — nor tool-agnostic exchange, locking developers into particular toolchains like LLVM.
+however current de-facto interchange formats (such as QIR and OpenQASM) either aren't designed for
+quantum optimization — lacking the explicit structure needed for advanced high-level
+optimizations — nor tool-agnostic exchange, locking developers into particular toolchains like
+LLVM.
```

Comment:
Suggest rephrasing since "quantum optimization" is ambiguous - I assume what is meant is quantum program optimization and not quantum algorithms for optimization. But also, I don't think it's the case that they aren't designed for compilation/optimization, it's more that the format is amenable only to optimization of the lowest-level (gate) instructions.

```suggestion
however current de-facto interchange formats (such as QIR and OpenQASM) either lack the explicit structure needed for advanced high-level
optimizations, or are not tool-agnostic, locking developers into particular toolchains like
LLVM.
```

Reply:
Good point, might make a slight modification here:

```suggestion
however current de-facto interchange formats (such as QIR and OpenQASM) either lack the explicit structure needed for advanced high-level
quantum optimizations, or are not tool-agnostic, locking developers into particular toolchains like
LLVM.
```